client/quantum_serverless/core/tracing.py

```diff
@@ -114,3 +114,17 @@ def _trace_env_vars(env_vars: dict, location: Optional[str] = None):
         env_vars[OT_TRACEPARENT_ID_KEY] = traceparent
         os.environ[OT_TRACEPARENT_ID_KEY] = traceparent
     return env_vars
+
+
+def setup_tracing() -> None:
+    """Setup Tracing for Ray cluster
+
+    Passed as an argument at Ray start
+    """
+    resource = Resource(attributes={SERVICE_NAME: "Quantum-Serverless: Ray"})
+    provider = TracerProvider(resource=resource)
+    otel_exporter = BatchSpanProcessor(
+        OTLPSpanExporter(endpoint="http://otel-collector:4317/", insecure=True)
```

Comment:
out of curiosity, would it cause any issues here if an alternative agent host/port is passed in L77 ? I assume those are separate so there's no reason to worry, but figure it doesn't hurt to ask

Reply:
oops,  it's for my debugging.  I'll change it to
```
f"{agent_host}:{agent_port}"
```
Thanks!